config/notifications.js

```diff
@@ -39,7 +40,30 @@ function sendPushNotification(tokens, alert, badgeCount, payload, category, dela
   }, delay);
   setTimeout(async () => {
     // Send the actual notification
-    sendPushNotificationMessage(tokens, notification);
+
+    //@TODO payload.title needs to be refactored! why do we have userName@channelName?????
+    const title = notification.payload.title;
+
+    if(!title){
+      throw new Error('title is not valid');
+    }
+
+    const body = notification.payload.message;
+
+    if(!body){
+      throw new Error('body is not valid');
+    }
+    console.log('*******', notification.payload);
+    if(!(notification.payload)){
+      throw new Error(' notification object is not properly formed')
+    }
+
+    const data = notification.payload.metadata
+
+    const message = firebaseService.generateMessage(title, body, data);
+    const options = firebaseService.generateOptions();
+    await firebaseService.sendPushNotification(tokens, message, options) //async sendPushNotification(registrationToken, message, options = null){
```

Comment:
@anggeloexe what happens if there is an exception error?

Reply:
@reyesrene  The process fails and the server restarts or stops. Doing tests I see that there is still no correct global error handling in Metis API